### ISSUE TITLE
D and O are too similar

### ISSUE THREAD

**Issue by david-gauquelin** [xants2355/snaha-regular#1]
<img width="258" alt="Screenshot 2024-11-28 at 16 48 59" src="https://github.com/user-attachments/assets/f496fce7-2d64-46e9-bce7-a179d895ff04">

In all caps the DO combination is really hard to read. And it's a quite frequent combination.

**Comment by xants2355** [comment]
You are right, but I think it is not critical, the other letters in words will help read and understand words. So, I would leave it as it is, if we deside that it should be corrected, I will do it.

**Comment by david-gauquelin** [comment]
So far it didn't cause a problem, I will close this issue now and reopen if it becomes an actual issue in use
